src/main/scss/components/_buttons.scss

```diff
@@ -75,6 +75,35 @@
   }
 }
 
+.jenkins-button--outline {
+  color: var(--btn-secondary-color) !important;
+  border: 2px solid var(--btn-secondary-border);
+  font-weight: bold;
+  font-size: var(--font-size-xs);
+  &::before {
+    background: transparent !important;
+  }
+
+  &:not(:disabled) {
+    &:hover {
+      border: 2px solid var(--accent-color) !important;
```

Comment:
This wouldn't work if paired with colour modifiers.

Reply:
hmm are you sure 
![image](https://github.com/user-attachments/assets/83fb3298-9047-4a2b-b8d8-904c08530f2f)
however I added `jenkins-button--outline[class*="color"]`

Reply:
`jenkins-button jenkins-button--outline jenkins-!-success-color`
![image](https://github.com/user-attachments/assets/23ce385e-f837-4b41-b01e-c93dfccd9f9f)
:hover
![image](https://github.com/user-attachments/assets/933e80c9-2590-48e6-8758-85edd8472d99)

Reply:
https://github.com/jenkinsci/jenkins/pull/10061/commits/6188712a6d8802d965ee4f9f75c6a6abd7509f3a

Reply:
Heya, I'm travelling today but I'll take a look as soon as I can :)

Reply:
good traveling and MCAAHNY

Reply:
Thanks, hope you had a good time too :)

<img width="360" alt="image" src="https://github.com/user-attachments/assets/8450e48c-4227-46a7-8c5a-e56bf3a2df31" />

```
.jenkins-button--outline {
  color: var(--color, var(--text-color-secondary)) !important;
  --button-background--hover: color-mix(in srgb, var(--color, var(--text-color-secondary)) 15%, transparent);
  --button-background--active: color-mix(in srgb, var(--color, var(--text-color-secondary)) 20%, transparent);
  --button-box-shadow--focus: color-mix(in srgb, var(--color, var(--text-color-secondary)) 10%, transparent);

  &::before {
    background: transparent;
    border: 2px solid var(--color, var(--text-color-secondary));
    opacity: 1 !important;
  }

  &:hover {
    &::before {
      background: var(--button-background--hover) !important;
    }
  }

  &:active {
    &::before {
      background: var(--button-background--active) !important;
    }
  }
}

.jenkins-button[class*="color"] {
  background: transparent;
  color: var(--color) !important;

  &::before {
    --button-background: currentColor; // changed this
    opacity: 0.1;
  }
```

How is something like this? The buttons styling is a little confusing (and more complex than it needs to be as modern CSS wasn't available when it was originally written, I do wanna tidy up this file)

I’m not totally sure about adding an outline button to `core` - I feel our default/secondary style already covers the same ground. Any thoughts on using it as a local class, using a prefix like `{plugin-name}-` instead of `jenkins-`? 

Any thoughts @jenkinsci/sig-ux ?